README.md

```diff
@@ -38,16 +38,16 @@ repositories {
 }
 ```
 
-Next add a dependency in the `build.gradle` file of your app module. The
+Next, add a dependency in the `build.gradle` file of your app module. The
 following will add a dependency to the full library:
 
 ```gradle
 implementation 'com.google.android.exoplayer:exoplayer:2.X.X'
 ```
 
 where `2.X.X` is your preferred version. Alternatively, you can depend on only
-the library modules that you actually need. For example the following will add
-dependencies on the Core, DASH and UI library modules, as might be required for
+the library modules that you actually need. For example, the following will add
+dependencies to the Core, DASH and UI library modules, as might be required for
```

Comment:
I'm not convinced about this change. You "depend on" rather than "depend to". The way it's phrased after this change implies that the Core, DASH and UI modules are actually being modified with new dependencies. Not sure the comma changes are really necessary either.